ethpm_cli/commands/auth.py

```diff
@@ -49,7 +130,7 @@ def get_authorized_address() -> Address:
     Returns the address associated with stored keyfile. No password required.
     """
     keyfile = get_keyfile_data()
-    return keyfile["address"]
+    return add_0x_prefix(keyfile["address"])
```

Comment:
thoughts on making this a checksum address?  It's getting logged in a bunch of places which means it will end up in people's clipboards.  Since people will need to fund this address, they're likely to copy/pasta it to send funds...